RyuSocks/Auth/UsernameAndPassword.cs

```diff
@@ -9,21 +9,23 @@ namespace RyuSocks.Auth
     [AuthMethodImpl(0x02)]
     public class UsernameAndPassword : IProxyAuth
     {
-        public int WrapperLength { get; }
+        // Set to 0, Since there is no wrapping happening here
```

Comment:
```suggestion
```

I'd just omit the comment here, since it should be obvious enough why the wrapper length is set to 0.